langwatch/src/pages/onboarding/organization.tsx

```diff
@@ -463,10 +448,10 @@ export default function OrganizationOnboarding() {
                     colorPalette="orange"
                     type={isSaaS ? "button" : "submit"}
                     disabled={
-                      createOrganization.isLoading ||
-                      createOrganization.isSuccess
+                      initializeOrganization.isLoading ||
+                      initializeOrganization.isSuccess
```

Comment:
Consider combining this into a memoized value

Reply:
I think memoization on this would actually degrade performance more as memoization isn't as cheap of an operation as a boolean comparison.